requirements/env_docs.yml

```diff
@@ -8,7 +8,6 @@ dependencies:
   - ipython
   - ipykernel
   - mccabe>=0.6
-  - nb_conda
```

Comment:
This is a bit unrelated to this PR imho. Why is this necessary for the tag removal?

Reply:
Right, this is only for making readthedocs run again. It's now in develop as well.